tests/zfs-tests/tests/functional/cli_root/zfs_clone/zfs_clone_010_pos.ksh

```diff
@@ -206,28 +202,23 @@ for ds in $datasets; do
 done
 names=$(zfs list -rt all -o name,clones $TESTPOOL)
 log_must verify_clones 3 1 $TESTCLONE
-i=1
 for ds in $datasets; do
 	log_must zfs promote $ds
-	((i=i+1))
 done
 log_must local_cleanup
 
 log_note "verify clone list truncated correctly"
-typeset -i j=200
-i=1
 fs=$TESTPOOL/$TESTFS1
+xs=$(python -c 'print("x" * 200)')
```

Comment:
i'd like reduce dependency to python and try to use changes based on SHELL

Reply:
The test runner is written in Python. Are you using something else to run tests?

Reply:
i know about runner, but right now we try to move from python2 to python3 and want update tests every time with migration?
also, tests execution can be done in another way instead of by `run` based on python and reduce hardcode to it can help.

Reply:
`for i in {0..200}; do xs=${xs}x; done`

Reply:
Since we don't need to have a python dependency here, I agree it would be nice to replace it with @richardelling's solution.